portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,12 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.link {
+  text-decoration: none;
+}
+
+.icon {
+  height: 20px;
+  width: 20px;
+}
```

Comment:
Ooh it's good practice to include a newline at the end of any file you're working on! https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file goes into this a little bit.

Reply:
Done!